.github/workflows/docker-build-image.yml

```diff
@@ -0,0 +1,198 @@
+# This workflow will build and push a new Docker image to the default Artifact Registry in SSB.
+#
+# Overview:
+# 1. Authenticate to Google Cloud
+# 2. Authenticate to Artifact Registry
+# 3. Build and publish a docker image to Google Artifact Registry
+# 
+# To configure this workflow:
+# 1. Change the values for the IMAGE_NAME and TEAM_NAME environment variables (below).
+# 2. The image will be tagged with the git commit hash and latest. To override the default tags, add tags to the build_metadata step.
+#
+# This workflow is tailored for SSB´s setup, however if you wish to use this in a non SSB context ensure/change the following:
+# 1. Create GitHub secrets for GAR_PROJECT_NUMBER and GAR_PROJECT_ID (Number and ID for the Artifact Registry project).
+# 2. Create and configure Workload Identity Federation for GitHub.
+
+name: Docker Image CI
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - "main"
+    paths: 
+      - 'docker/**'
+      - '.github/workflows/docker-build-image.yml'
+  pull_request:
+    paths: 
+      - 'docker/**'
+      - '.github/workflows/docker-build-image.yml'
+
+env:
+  REGISTRY: europe-north1-docker.pkg.dev/${{ secrets.GAR_PROJECT_ID }}
+  IMAGE_NAME: rstudio-onprem  # TODO: set this to your preferred image name
+  TEAM_NAME: itinfra   # TODO: set this to you teams name
```

Comment:
The team name here is wrong which makes the authentication step point to a service account that doesn't exist
```diff
-itinfra
+itinfra-ks
```